osgtest/tests/test_550_condorce.py

```diff
@@ -64,69 +83,55 @@ def run_blahp_trace(self, lrms):
         # S'2'
         # p6
         # s."
-        self.assert_(re.search(r'BatchJobId[=\s"\'p1S]+%s' % backend_jobid, cache),
-                     'Job %s not found in %s blahp cache:\n%s' % (backend_jobid, lrms.upper(), cache))
+        self.assertTrue(re.search(r'BatchJobId[=\s"\'p1S]+%s' % backend_jobid, cache),
+                        'Job %s not found in %s blahp cache:\n%s' % (backend_jobid, lrms.upper(), cache))
 
         os.chdir(cwd)
 
-    def general_requirements(self):
-        core.skip_ok_unless_installed('condor', 'htcondor-ce', 'htcondor-ce-client')
-        self.skip_bad_unless(service.is_running('condor-ce'), 'ce not running')
-
     def test_01_status(self):
-        self.general_requirements()
-
         command = ('condor_ce_status', '-any')
         core.check_system(command, 'ce status', user=True)
 
     def test_02_queue(self):
-        self.general_requirements()
-
         command = ('condor_ce_q', '-verbose')
         core.check_system(command, 'ce queue', user=True)
 
     def test_03_ping(self):
-        self.general_requirements()
-        self.skip_bad_unless(core.state['proxy.valid'], 'requires a proxy cert')
-
-        command = ('condor_ce_ping', 'WRITE', '-verbose')
-        stdout, _, _ = core.check_system(command, 'ping using GSI and gridmap', user=True)
-        self.assert_(re.search(r'Authorized:\s*TRUE', stdout), 'could not authorize with GSI')
+        self.check_write_creds()
+        self.command += ['condor_ce_ping', 'WRITE', '-verbose']
+        stdout, _, _ = core.check_system(self.command, 'ping using GSI and gridmap', user=True)
+        self.assertTrue(re.search(r'Authorized:\s*TRUE', stdout), 'could not authorize with GSI')
```

Comment:
```suggestion
        stdout, _, _ = core.check_system(self.command, 'ping', user=True)
        self.assertTrue(re.search(r'Authorized:\s*TRUE', stdout), 'could not authorize')
```
since it's not necessarily GSI anymore